notebooks/nuextract-structure-extraction/nuextract-structure-extraction.ipynb

```diff
@@ -0,0 +1,824 @@
+{
```

Comment:
<p>we follow the rule to use the latest released version of our tools as lower bound, please update nncf:</p><p>nncf&gt;=2.12.</p><p> </p><p>I do not see datasets usage in notebook, please remove.</p><p> </p><p>Also optimum and onnx are dependencies of optimum-intel, there is no need to specify them explicitly and allow optimum-intel select required versions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>